aict_tools/scripts/split_data.py

```diff
@@ -97,17 +101,19 @@ def split_multi_telescope_data(input_path, output_basename, fraction, name):
         selected_array_events = array_events[array_events.run_id.isin(selected_run_ids)]
         selected_telescope_events = telescope_events[telescope_events.run_id.isin(selected_run_ids)]
 
-        path = output_basename + '_' + part_name + '.hdf5'
+        path = output_basename + '_' + part_name + file_extension
         log.info('Writing {} runs events to: {}'.format(n, path))
-        write_data(selected_runs, path, key='runs', use_h5py=True, mode='w')
-        write_data(selected_array_events, path, key='array_events', use_h5py=True, mode='a')
-        write_data(selected_telescope_events, path, key='telescope_events', use_h5py=True, mode='a')
+        write_hdf(selected_runs, path, table_name='runs', use_h5py=use_h5py, mode='w')
+        write_hdf(selected_array_events, path, table_name='array_events', use_h5py=use_h5py, mode='a')
+        write_hdf(selected_telescope_events, path, table_name='telescope_events', use_h5py=use_h5py, mode='a')
         log.debug(f'selected runs {set(selected_run_ids)}')
         log.debug(f'Runs minus selected runs {ids - set(selected_run_ids)}')
         ids = ids - set(selected_run_ids)
 
 
-def split_single_telescope_data(input_path, output_basename, fmt, inkey, key, fraction, name):
+def split_single_telescope_data(input_path, output_basename, inkey, key, fraction, name, use_h5py=True):
+
+    _, file_extension = os.path.splitext(input_path)
 
     if fmt in ['hdf5', 'hdf', 'h5']:
```

Comment:
This check is performed in io/read_data now (.csv files don't seem to be supported with the new read_data method tho)